frontend/packages/knative-plugin/src/components/routes/RouteDetailsPage.tsx

```diff
@@ -1,21 +1,74 @@
 import * as React from 'react';
+import { ClipboardCopy } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
 import { useActivePerspective } from '@console/dynamic-plugin-sdk';
-import { DetailsForKind } from '@console/internal/components/default-resource';
+import { Conditions } from '@console/internal/components/conditions';
 import { DetailsPage } from '@console/internal/components/factory';
-import { navFactory } from '@console/internal/components/utils';
+import { RoutesDetailsProps } from '@console/internal/components/routes';
+import {
+  detailsPage,
+  ExternalLink,
+  navFactory,
+  ResourceSummary,
+  SectionHeading,
+} from '@console/internal/components/utils';
 import { K8sKind, K8sResourceKind, referenceForModel } from '@console/internal/module/k8s';
 import {
   ActionMenu,
   ActionMenuVariant,
   ActionServiceProvider,
   useTabbedTableBreadcrumbsFor,
 } from '@console/shared';
+import { PRIVATE_KNATIVE_SERVING_LABEL } from '../../const';
 import { serverlessTab } from '../../utils/serverless-tab-utils';
 
+const RouteDetails: React.FC<RoutesDetailsProps> = ({ obj: route }) => {
+  const { t } = useTranslation();
+  const isPrivateKSVC =
+    route?.metadata?.labels?.[PRIVATE_KNATIVE_SERVING_LABEL] === 'cluster-local';
+  return (
+    <>
+      <div className="co-m-pane__body">
+        <SectionHeading text={t('knative-plugin~Route details')} />
+        <div className="row">
+          <div className="col-sm-6">
+            <ResourceSummary resource={route} />
+          </div>
+          <div className="col-sm-6">
+            <dl className="co-m-pane__details">
```

Comment:
Better to use `DetailsItem` component instead of `dl` tag.

Reply:
Hi @vikram-raj / @Lucifergene ,

Updated PR to add `DetailsItem` component instead of `dl` tag. PTAL.